From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Paul Praschl <praschlpaul@g-p.at>
Date: Tue, 4 Aug 2026 18:08:42 +0200
Subject: [PATCH 106/117] arm64: dts: apple: Set D111 BCM4355 board type

Publish the olaf Wi-Fi module identifier so brcmfmac selects D111
firmware using the module board type instead of the root device
compatible.

Signed-off-by: Paul Praschl <praschlpaul@g-p.at>

Orchard-replay-of: 335328820d717ba5196300b1e69a38b831418df9
---
 arch/arm64/boot/dts/apple/t8010-d111.dts | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/boot/dts/apple/t8010-d111.dts b/arch/arm64/boot/dts/apple/t8010-d111.dts
index aed2cbf3e..57bde8f15 100644
--- a/arch/arm64/boot/dts/apple/t8010-d111.dts
+++ b/arch/arm64/boot/dts/apple/t8010-d111.dts
@@ -56,6 +56,7 @@
 			device-wake-gpios = <&pinctrl_ap 88 GPIO_ACTIVE_HIGH>;
 			vdd-supply = <&wlan_vdd>;
 			apple,antenna-sku = "XX";
+			brcm,board-type = "apple,olaf";
 		};
 	};
 };
